extensions/wikia/Listusers/SpecialListusers_helper.php

```diff
@@ -185,13 +176,12 @@ public function loadData() {
 				$sk = $context->getSkin();
 				/* select records */
 				$oRes = $dbs->select(
-					array( self::TABLE . ( ($this->mUseKey) ? ' use key('.$this->mUseKey.')' : '' ) ),
+					array( self::TABLE ),
 					array(
 						'user_id',
 						'cnt_groups',
 						'all_groups',
 						'edits',
-						'user_is_blocked',
```

Comment:
Does that mean that we can drop this column?

Reply:
@macbre After this goes live and backend scripts stop inserting to this column, yes. IMO it makes no sense to keep this column, since you can't listen to the event of a block expiring - you'd have to maintain cron job just for that. Better to check if the user is blocked when rendering ListUsers.

Reply:
@macbre — you can drop whatever you wish!